frontends/web/src/api/account.ts

```diff
@@ -105,7 +113,7 @@ export type TAccountsTotalBalanceResponse = {
     totalBalance: TAccountsTotalBalance;
 } | {
     success: false;
-    errorCode?: 'ratesNotAvailable';
+    errorCode?: string;
     errorMessage?: string;
```

Comment:
Remove these too since they are unused (also in the backend then)?

Reply:
Done

Reply:
Yup missed that, done!